software/IOIOLibAndroid/build.gradle

```diff
@@ -18,4 +18,5 @@ android {
 
 dependencies {
     api project(":IOIOLibCore")
+    implementation 'androidx.appcompat:appcompat:1.2.0'
```

Comment:
That's causes an error in sample app !

But we use 

```
dependencies {
    implementation 'com.github.ytai.ioio:IOIOLibAndroid:5.07'
    ...
}
```

That's why we don't see it.

The reason, the consumer needs now this library as well